### DISCUSSION TITLE
tool/aliyundrive/callback

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#165] [Announcements]
# tool/aliyundrive/callback

import Callback from 

https://alist.nn.ci/tool/aliyundrive/callback.html?code=f478908f0a384bba829a8b30bebea7b2

**Comment by LMS-potato** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiI2YWNjOTFmYjZkOWM0YzQ0ODY3MWRmOGUwMjgyNjBmOSIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1MDY0Nzg1LCJpYXQiOjE2NzcyODg3ODV9.k7As1t_UnCJcIChmYRKp735KfXo_sAGEsY0sqf2WeekgIAFQRs-hj-jVYiieumEcv3PwFcSTVSU8hxfijlUkrQ

**Comment by woydlwo** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiI2YWNjOTFmYjZkOWM0YzQ0ODY3MWRmOGUwMjgyNjBmOSIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1MDY0Nzg1LCJpYXQiOjE2NzcyODg3ODV9.k7As1t_UnCJcIChmYRKp735KfXo_sAGEsY0sqf2WeekgIAFQRs-hj-jVYiieumEcv3PwFcSTVSU8hxfijlUkrQ

**Comment by yuanpenghe** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiI4Y2U2ZjJkY2JmMmI0YmUwODcxNzAwYTFmMTAxM2M0MiIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1MTEwMzE0LCJpYXQiOjE2NzczMzQzMTR9.VtMSzxkeBihdwMSB60IIR1Dbyy4zInEfSudjV1y4xkmIQnLBS9xJBjkH7op5VHAC0gxNUnCXwT_D0SlaUrnbBQ

**Comment by conquhare** [comment]
Failed init storage: failed to refresh token: The input parameter refresh_token is not valid.

**Comment by Timber552** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiI5YTE3NDFiMWZhODE0YjI2OTA5ZTJjNzFiNmViMjZkZSIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1MTQ3MTczLCJpYXQiOjE2NzczNzExNzN9.J9fX51Azp6X3iYouIQcOZyogK7myAEkoqUUZunPgZeaAkffZM_oXTAuP5OY83T7fyB9qBXFKsU-dpYESZHKWcQ

**Comment by jasnxu** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiI2MGIxYjk4ZmRiZmU0ZGRiYmJjYTI4YjQ5YjRlNTUxNSIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1MTU0NzU5LCJpYXQiOjE2NzczNzg3NTl9.iwAdY4_fXmuXaDxxr5u72bV9OtHANsOdIwDYS8eVbmcKs9Fd_1ZRK8JW6ZqAH1GchNyT9zMEjqfBSzs3C-V3lQ

**Comment by yjs0370** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJmY2RjNTc4ZmI5ZDk0Y2U4OWE2ZTFmNDkwOGY0NTcyMyIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1MjUxODkxLCJpYXQiOjE2Nzc0NzU4OTF9.fWGUyRs-9O0H91elYFXe65r_wpPAVAVo0NeRXE95FIA7xyUrgI2CFLkCcBj9sOU949AMBNWpSHwR_7AKBVKTVw

**Comment by gaochengwen** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJjODFhMWFlYjNiZjE0YTUwOGI4NmU4ODZlYmUxZmVjMyIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1MzUxMjYzLCJpYXQiOjE2Nzc1NzUyNjN9.Sn0DVqTHB5n1ZAiy01FISS4ygb3vMg-O4KI8HkM_3unGDY2AT8B-c02a7p1vcph7zO8vXjKYqbTB0zLF2W8pgA

**Comment by Newley0328** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiIzNTU5ZmZjYTYxYzE0YTI2ODk1ODMwMzE3MzYzOGY3NCIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjo
xNjg1MzUzMjE5LCJpYXQiOjE2Nzc1NzcyMTl9.UhRl7twhtqCVeYkmkhSKI0vwmz1sMnZFYdDwJhnfBvp_AgPUCyVmFJGx8sJLgt6LEtleA-LAxGNtMTXXC0a6zQ

**Comment by Newley0328** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiIzNTU5ZmZjYTYxYzE0YTI2ODk1ODMwMzE3MzYzOGY3NCIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1MzUzNzMxLCJpYXQiOjE2Nzc1Nzc3MzF9.DS4cUW4F8LsZnjIM3acmldwPXKbNRVUmn1rSe35jsp_AhZq767sMTQWO5sbjPKPxWI8x_m6xMmd_uxljT3TcLg

**Comment by dcshow** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiI4N2VhYmFkNzAxN2I0ODBlODM4ZDdkNTlmZDJkNjJkMCIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1MzczODc5LCJpYXQiOjE2Nzc1OTc4Nzl9.hfy3ExNY3f1O7cSiS-P33SN5KMxuSDHFLYkhtQ09zYAF3RS2axcRKshmPtEOhmczdX4rL95q9Uq5pxRa2ceViA

**Comment by terrence2u** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiI2ZDE0MmYyNGJiYWI0OTRmODRmMDNkNmM3NjgzNmY5NyIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1MzgwMjE2LCJpYXQiOjE2Nzc2MDQyMTZ9.v3bbs7eATU61KMg2NmkG8k1eosQ0Q0QJssCXPuXepANz3WCroF9AP8-TDzWRILX6lmV-21VC2NMrDj2jL-AAlw

**Reply by Jio0oiJ** [reply]
Failed init storage: Post "https://api.nn.ci/alist/ali_open/token": read tcp **********:34246->***********52:443: read: connection reset by peer

**Comment by Jio0oiJ** [comment]
Failed init storage: Post "https://api.nn.ci/alist/ali_open/token": read tcp **********:34246->***********52:443: read: connection reset by peer

**Comment by wtflfqlfq** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiIxYTAxZjhmNmVlMGY0ZGEyYmMxYzQ5MWE5ZDgzMWI4ZiIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1NDM1OTAyLCJpYXQiOjE2Nzc2NTk5MDJ9.OO7PxM1mBCQ_VWlzUtHpbjwwvF7TRRpXlugi35oP87IR3iMCTsGeeL967NXr8aHq7IjjHrHfHfLuRVRwtBI7Jw

**Comment by whylx222** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiIzNzMxODJiZDkyYzU0MGFiOTQ3YmI0YmEwNmI1YTAzZCIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1NDM5MTUyLCJpYXQiOjE2Nzc2NjMxNTJ9.RQ4Xe9MLnWnfjcSWbusB55-zm7MC20mMukTAKrS8N8C-EPlLEoRoGa0KuSwl2e7bQzAtwywV59NToZ0NwOzuzg

**Comment by whylx222** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiIzNzMxODJiZDkyYzU0MGFiOTQ3YmI0YmEwNmI1YTAzZCIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1NDM5NTkyLCJpYXQiOjE2Nzc2NjM1OTJ9.BPNrD5wl8-R2JY1fjYF48hqcQLKiuCxyW1aoRLv-Mwt2aWiV9ABJwet4XWtvzJ5S22EBrkDImyiizIAFL3Izrw

**Comment by whylx222** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiIzNzMxODJiZDkyYzU0MGFiOTQ3YmI0YmEwNmI1YTAzZCIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1NDQwMzIwLCJpYXQiOjE2Nzc2NjQzMjB9.MJy5ViNOhzTK6NXK04X4zbI54DxkKcDzvSA7V2OOKxYsl-YHRalNRG5SFzHsXRK4XmyfxCP2aXhEMt-WQcMf-g

**Comment by zgsouth** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJjZTAwYjgxN2NmYTE0NWZmODY3NDlkZjg2YmM5MDIyYSIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1NDQ5MzI2LCJpYXQiOjE2Nzc2NzMzMjZ9.grPqRktp_w5dM59fk5eRhYbUhrf9hi2hxoPTvNrRqNBq1F9FEd2XdKW-wKggUo5NB3Y_0wwKTOLHZZYbZfc9aQ

**Comment by zgsouth** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJjZTAwYjgxN2NmYTE0NWZmODY3NDlkZjg2YmM5MDIyYSIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1NDQ5NjM5LCJpYXQiOjE2Nzc2NzM2Mzl9.lSYUz0p5TgOONUrbTt4pXN2bmwMuQq-pGVqN9WGhJ5YN9CU9SthHOhb0-_WkOLqbZnWCYEvKAPjvt50SJVrWAA

**Comment by unhappym55** [comment]
eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiI5Y2M0NDJjZGM5MmQ0YTJhYjU0MGM4NzA4MmFjNmE4OSIsImF1ZCI6Ijc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmIiwiZXhwIjoxNjg1NDYxOTEyLCJpYXQiOjE2Nzc2ODU5MTJ9.e8ti9LA7xkJMkm9TyMk6r528G4F50Vrsnbd6EAZg3Uep4VLovPttOGkC9Kxjwy-zNA6M4QeXZ0E5rjnH-bjCtg
